ostree/ostree_dest.go

```diff
@@ -131,6 +131,7 @@ func (d *ostreeImageDestination) importBlob(blob *blobToImport) (error) {
 		return err
 	}
 	return exec.Command("ostree", "commit",
+		fmt.Sprintf("--repo=%s", d.ref.repo),
```

Comment:
BTW `ostree` seems to also accept separate parameters without the `=`e.g. `[…, "--repo`, d.ref.repo, …]`. That would allow getting rid of all of these `fmt.Sprintf` calls.